LiskHQ/lisk-hub
# Fix see all transactions - Closes #571

### What was the problem?
`See all transactions` didn't redirect to`/transactions`
### How did I fix it?
Changed route on `See all transactions`
### How to test it?
Go to dashboard and click `See all transactions` it should redirect you to `/transactions`
### Review checklist
- The PR solves #571
- All new code is covered with unit tests
- All new features are covered with e2e tests
- All new code follows best practices
